stable/pipelines/templates/pipelines-statefulset.yaml

```diff
@@ -22,6 +22,22 @@ spec:
         component: {{ include "pipelines.services.name" . }}
       annotations:
         checksum/systemyaml: {{ include (print $.Template.BasePath "/pipelines-system-yaml.yaml") . | sha256sum }}
+        checksum/secretdb: {{ include (print $.Template.BasePath "/database-secret.yaml") . | sha256sum }}
+        {{- if and .Values.buildPlane.dynamic.provider.aws.enabled ( not .Values.buildPlane.dynamic.provider.aws.existingSecret ) }}
+        checksum/secretaws: {{ include (print $.Template.BasePath "/buildplane-secret-aws.yaml") . | sha256sum }}
+        {{- end }}
+        {{- if and .Values.buildPlane.dynamic.provider.aws.enabled }}
+        checksum/configaws: {{ include (print $.Template.BasePath "/buildplane-config-aws.yaml") . | sha256sum }}
+        {{- end }}
+        {{- if and .Values.buildPlane.dynamic.provider.k8s.enabled ( not .Values.buildPlane.dynamic.provider.k8s.existingSecret ) }}
+        checksum/secretk8s: {{ include (print $.Template.BasePath "/buildplane-secret-k8s.yaml") . | sha256sum }}
+        {{- end }}
+        {{- if and .Values.buildPlane.dynamic.provider.k8s.enabled }}
```

Comment:
The and here is redundant

Reply:
I have removed all conditions for the checksum